### PR TITLE
docs: align getting-started setup to reflect create-tauri-app v3

### PR DESCRIPTION
**Pull request by unvalley**
According to the [create-tauri-app Version 3 Released](https://tauri.app/blog/2023/03/01/create-tauri-app-version-3-released/), `create-tauri-app` will not provide templates for nextjs and sveltekit. So, we should remove the `create-tauri-app` guide from docs for those templates.

### CODE REVIEW

File: docs/guides/getting-started/setup/_fragments/_create-tauri-app-guide.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/_fragments/_create-tauri-app-guide.mdx]:
```suggestion
The easiest way to scaffold a new project is the [`create-tauri-app`] utility. It provides opiniated templates for vanilla HTML/CSS/JavaScript and many frontend frameworks like React, Svelte and Yew.
```

Using this as a chance to remove a few Oxford commas

File: docs/guides/getting-started/setup/next-js.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/next-js.mdx]: The spaces after the headings are intentional so that the line below is indeed a newline (see the deploy preview for this page.

File: docs/guides/getting-started/setup/next-js.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/next-js.mdx]: Don't remove spaces

File: docs/guides/getting-started/setup/sveltekit.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/sveltekit.mdx]: Don't remove spaces

File: docs/guides/getting-started/setup/vite.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/vite.mdx]: Don't remove spaces

File: docs/guides/getting-started/setup/_fragments/_intro.mdx
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/_fragments/_intro.mdx:27]: Need to keep this link in here so that the link in line 3 isn't broken

**Review by lorenzolewis — CHANGES_REQUESTED** [review]
Thanks for putting this in, @unvalley! Just a few changes requested.

**Comment by FabianLars** [comment]
still super unsure about this change. cta is still somewhat of an alternative, even though it doesn't have _matching_ templates (iirc it didn't have the nextjs template when it was added to the docs either), so maybe a rewording is better? idk

Or maybe we just go with your PR and call it a day, and then for the new docs we can rethink our whole approach.

**Comment by lorenzolewis** [comment]
I'm on the side of let's merge this one in (once updated) and call it a day. The convos of updating it all will still be a bit so would rather have this out there to help clear up any confusion right now.

**Comment by lorenzolewis** [comment]
@unvalley just checking to make sure you know there's a few more requested changes to put in when you have a chance before merging in.

**Comment by unvalley** [comment]
@lorenzolewis Yes, I'll do it later. (I was on duty).

**Comment by lorenzolewis** [comment]
No rush at all, just wasn't sure if you noticed it. Thanks again! <3

File: docs/guides/getting-started/setup/_fragments/_intro.mdx
Comment (unvalley) [reply docs/guides/getting-started/setup/_fragments/_intro.mdx:27]: @lorenzolewis Thanks, I left the `prerequisites` link here, but I removed the `cargo` link on `_create-tauri-app-guide.mdx` because I think the link isn't used on the page.

**Comment by unvalley** [comment]
@lorenzolewis I fixed it! Please take a look.

**Review by lorenzolewis — APPROVED** [review]

**Comment by lorenzolewis** [comment]
Looks good to me, thanks a bunch for putting this together!

**Pull request merged**
